### PR TITLE
feat: add ZOOMER

### PR DESCRIPTION
**Pull request by rhlsthrm**
**Description**

Add Optimism and Base configs for Zoomer.

**Tests**

Please describe any tests you've added. If you've added no tests, or left important behavior untested, please explain why not.

**Additional context**

Add any other context about the problem you're solving.

**Metadata**

- Fixes #[Link to Issue]

### CODE REVIEW

**Comment by rhlsthrm** [comment]
These are not tokens that are minted from the OP bridge by the way, that's why the validation is failing I believe.

**Comment by rhlsthrm** [comment]
cc @roberto-bayardo

**Comment by roninjin10** [comment]
Might need to add a nobridge: true property if they aren't Optimism standard tokens

**Comment by rhlsthrm** [comment]
Thanks @roninjin10 I missed this in the readme. Just modified to add the property.

**Comment by hamdiallam** [comment]
Hi! The validation script needs to pass. It looks like the logo must be of filename `logo.png`. Otherwise looks good on OP side and needs a base approval

**Comment by cfluke-cb** [comment]
"name": "ZoomerCoin", should be "name": "ZOOMER"

**Comment by cfluke-cb** [comment]
Hi @rhlsthrm This is still not passing validation. I am currently getting errors
error: ZOOMER on chain optimism token 0xB962150760F9A3bB00e3E9Cf48297EE20AdA4A33 has incorrect symbol
error: ZOOMER on chain optimism token 0xB962150760F9A3bB00e3E9Cf48297EE20AdA4A33 has incorrect name. Got xZoomerCoin
error: ZOOMER on chain base token 0xD1dB4851bcF5B41442cAA32025Ce0Afe6B8EabC2 has incorrect name. Got Based ZoomerCoin 

Locally you can run 
npx tsx ./bin/cli.ts validate --datadir ./data --tokens ZOOMER
to test this yourself

**Comment by rhlsthrm** [comment]
OK I see the problem here, it's that the token has different names and symbols on different chains. Should I break them into separate folders?

**Comment by rhlsthrm** [comment]
Btw, I can't run the validate script locally, I get this error:

```
npx tsx ./bin/cli.ts validate --datadir ./data --tokens ZOOMER
(node:90737) ExperimentalWarning: `--experimental-loader` may be removed in the future; instead use `register()`:
--import 'data:text/javascript,import { register } from "node:module"; import { pathToFileURL } from "node:url"; register("file%3A///Users/rahul/dev/projects/zoomer/ethereum-optimism.github.io/node_modules/.pnpm/tsx%403.12.6/node_modules/tsx/dist/loader.js", pathToFileURL("./"));'
(Use `node --trace-warnings ...` to show where the warning was created)
[ 'ZOOMER' ]
node:fs:453
    return binding.readFileUtf8(path, stringToFlags(options.flag));
                   ^

Error: ENOTDIR: not a directory, open 'data/.DS_Store/data.json'
    at Object.readFileSync (node:fs:453:20)
    at <anonymous> (/Users/rahul/dev/projects/zoomer/ethereum-optimism.github.io/src/generate.ts:35:12)
    at Array.map (<anonymous>)
    at generate (/Users/rahul/dev/projects/zoomer/ethereum-optimism.github.io/src/generate.ts:33:6)
    at validate (/Users/rahul/dev/projects/zoomer/ethereum-optimism.github.io/src/validate.ts:328:16)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at Command.<anonymous> (/Users/rahul/dev/projects/zoomer/ethereum-optimism.github.io/bin/cli.ts:25:21) {
  errno: -20,
  code: 'ENOTDIR',
  syscall: 'open',
  path: 'data/.DS_Store/data.json'
}

Node.js v20.11.1
```

**Comment by rhlsthrm** [comment]
I tried breaking it up based on the output you pasted, apologies for I can't run the test scripts locally for some reason.

**Comment by rhlsthrm** [comment]
In my latest commit I removed Ethereum from the chains because the Ethereum and Base tokens are named differently. How can I handle this case properly?

**Comment by cfluke-cb** [comment]
Hi @rhlsthrm You can override the names and symbols like this: https://github.com/ethereum-optimism/ethereum-optimism.github.io/blob/e1c95f902eb0d5a12df2e5db0bfebf450835017c/README.md?plain=1#L174

File: data/ZOOMER/data.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  "nobridge": true,
+  "description": "The first and only cross-chain memecoin for Gen Z",
+  "website": "https://zoomer.money",
+  "twitter": "@ZoomerCoin",
+  "tokens": {
+    "ethereum": {
+      "address": "0x0D505C03d30e65f6e9b4Ef88855a47a89e4b7676",
+      "overrides": {
+        "name": "ZoomerCoin",
+        "symbol": "ZOOMER"
```
Comment (wbnns) [inline data/ZOOMER/data.json:14]: This can be dropped since the symbol is already defined on Line 3.

**Review by wbnns — CHANGES_REQUESTED** [review]
@rhlsthrm 

Thanks for the updates; requested an additional change.

**Review by wbnns — CHANGES_REQUESTED** [review]
@rhlsthrm 

Thanks for the update. Can you also please add the `"nonstandard": true,` option to your JSON?

**Comment by rhlsthrm** [comment]
@wbnns done

**Review by wbnns — APPROVED** [review]
ACK

**Review by hamdiallam — APPROVED** [review]

**Pull request merged**